src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,26 +1,23 @@
+import { IHttp } from '../accessors/IHttp';
 import { IRead } from '../accessors/index';
-import { IRoom } from '../rooms/index';
-import { IUser } from '../users/index';
 import { IMessage } from './IMessage';
 
 export interface IPostMessageSentHandler {
     /**
      * First step when a handler is executed: Enables the handler to signal
-     * to the Rocketlets framework whether the it shall actually execute for the message
+     * to the Rocketlets framework whether handler shall actually execute for the message
      * about to be sent.
      *
      * @param message The message which is being sent
      * @param read An accessor to the environment
      * @return true: run the pre-logic
      */
-    isMessageApplicable(message: IMessage, read: IRead): boolean;
+    checkPostMessageSent(message: IMessage, read: IRead): boolean;
 
     /**
      * Method called *after* the message is sent to the other clients.
      *
-     * @param user The user who sent the message
-     * @param room The room where the message was sent to
      * @param message The message which was sent
      */
-    postMessageSent(user: IUser, room: IRoom, message: IMessage): void;
+    communicatePostMessageSent(message: IMessage, read: IRead, http: IHttp): void;
```

Comment:
@graywolf336 I thought about that and came to the conclusion, that it has almost no benefits of having a more global scope (see my paragraphs added in the brain dump). Particularly modifying accessors or those to very expensive or security related APIs should be injected. Only this way, their usage can effectively be controlled. 

=> Inject the `IHttp` (or a wrapper for "external ressources" which might also include file system access) to all methods where that's needed. If it was "all-but-one", it was worth it.